src/assets/css/footer.scss

```diff
@@ -0,0 +1,206 @@
+//Prueba para tener contenido y ver el footer al final
+body > p {
```

Comment:
put the height of the body or use a Lorem ipsum text. Please avoid comments in spanish.

Reply:
Done